### PR TITLE
feat: add 404 Error page

### LINKED ISSUE #16: Develop 404 Page

### PR DESCRIPTION
**Pull request by navaneethvenu**
- add 404 Error Page

Todo:
- Crosscheck Copies

Screenshots:

### CODE REVIEW

File: public/images/moon.svg
Comment (aurora-0025) [inline public/images/moon.svg:1]: something wrong with this svg?

**Review by aurora-0025 — DISMISSED** [review]

**Review by aurora-0025 — COMMENTED** [review]
Is this supposed to go to main branch?

**Comment by navaneethvenu** [comment]
> Is this supposed to go to main branch?

@aurora-0025 nope, added there by mistake. changed.

File: public/images/moon.svg
Comment (navaneethvenu) [reply public/images/moon.svg:1]: Yes, the header doesn't use this svg anymore. Will remove both sun and moon directly from dev (not in this pr)

**Review by aurora-0025 — APPROVED** [review]

**Pull request merged**